src/hera/task.py

```diff
@@ -170,6 +170,9 @@ class Task:
         that are set.
     image: str = 'python:3.7'
         The image to use in the execution of the function.
+    image_pull_policy: str = 'IfNotPresent'
```

Comment:
There's an `ImagePullPolicy` object available [here](https://github.com/argoproj-labs/hera-workflows/blob/main/src/hera/image.py#L4)! What do you think about making this default to `Always`? That's the default of Argo Workflows and I think there are many use cases when the `latest` tag is used, which would not be captured by `IfNotPresent`.